### PR TITLE
Adding MIT Licence

### PR DESCRIPTION
**Pull request by elsamuko**
The pypi package is MIT licenced: according to https://pypi.org/project/apt-repo/
This PR adds a LICENCE file

### CODE REVIEW

**Comment by elsamuko** [comment]
PS: I don't know why this PR lists the old commits, despite me having merged your master :/

**Comment by elsamuko** [comment]
@brennerm  
Hello, any update, if the LICENCE file can be merged?

Regards,
Samuel

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 brennerm@github
```
Comment (brennerm) [inline LICENCE]: @elsamuko Please change year to 2023 and name to _Max Brenner_

File: LICENCE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (brennerm) [inline LICENCE]: @elsamuko Please rename file to LICEN**S**E

**Comment by brennerm** [comment]
@elsamuko Sorry for the delay, struggling to find time for my OS projects these days. :/
Left 2 small review comment that I'd ask to you to incorporate.

File: LICENCE (hunk 1 above)
Comment (elsamuko) [reply LICENCE]: Done

File: LICENCE (hunk 2 above)
Comment (elsamuko) [reply LICENCE]: Done

**Comment by elsamuko** [comment]
> @elsamuko Sorry for the delay, struggling to find time for my OS projects these days. :/ Left 2 small review comment that I'd ask to you to incorporate.

Sure, no problem :)
I'm happy that your project exists.

**Comment by brennerm** [comment]
Thanks @elsamuko for your patience :)

**Pull request merged**